pkg/kn/traffic/compute.go

```diff
@@ -234,7 +238,7 @@ func verifyInputSanity(trafficFlags *flags.Traffic) error {
 
 	// equivalent check for `cmd.Flags().Changed("traffic")` as we don't have `cmd` in this function
 	if len(trafficFlags.RevisionsPercentages) > 0 && sum != 100 {
-		return errors.New(fmt.Sprintf("given traffic percents sum to 80, want 100"))
+		return errors.New(fmt.Sprintf("given traffic percents sum to %d, expected 100", sum))
```

Comment:
@zhanggbj : I went ahead and took this change as review comment to fix typo and included this in my original PR, hope its fine.

Reply:
@navidshaikh Sure